src/lib.rs

```diff
@@ -1081,6 +1161,111 @@ impl<T, S> IntoIterator for HashBag<T, S> {
     }
 }
 
+// ======== entry type
+#[cfg(feature = "amortize")]
+pub(crate) mod entry {
+    use griddle::hash_map::Entry;
+
+    #[derive(Debug)]
+    pub(crate) struct ForiegnEntry<'a, T, S> {
+        pub(crate) entry: Entry<'a, T, usize, S>,
+    }
+
+    impl<'a, T, S> ForiegnEntry<'a, T, S> {
+        pub(crate) fn new(entry: Entry<'a, T, usize, S>) -> Self {
+            Self {
+                entry,
+            }
```

Comment:
```suggestion
            Self { entry }
```
To make rustfmt happy